modules/ossm-operatorhub-install.adoc

```diff
@@ -0,0 +1,168 @@
+
+[WARNING]
+====
+Do not install Community versions of the Operators. Community Operators are not supported.
+====
+
+.Prerequisites
+* Access to the {product-title} web console.
+* The Elasticsearch Operator must be installed.
+
+.Procedure
+
+. Log into the {product-title} web console.
+
+. Navigate to *Catalog* -> *OperatorHub*.
+
+. Type *Jaeger* into the filter box to locate the Jaeger Operator.
+
+. Click the *Jaeger Operator* provided by Red Hat to display information about the Operator.
+
+. Click *Install*.
+
+. On the *Create Operator Subscription* page, select *All namespaces on the cluster (default)*. This installs the Operator in the default `openshift-operators` project and is made available to all projects in the cluster.
+
+. Select the *stable* Update Channel.
+
+. Select the *Automatic* Approval Strategy.
++
+[NOTE]
+====
+The Manual approval strategy requires a user with appropriate credentials to approve the Operator install and subscription process.
+====
+
+. Click *Subscribe*.
+
+. The *Subscription Overview* page displays the Jaeger Operator's install process.
+
+
+[id="ossm-operator-install-kiali_{context}"]
+== Installing the Kiali Operator
+You must install the Kiali Operator for the {ProductName} Operator to install the control plane.
+
+[WARNING]
+====
+Do not install Community versions of the Operators. Community Operators are not supported.
+====
+
+
+.Prerequisites
+
+* Access to the {product-title} web console.
+
+.Procedure
+
+. Log into the {product-title} web console.
+
+. Navigate to *Catalog* -> *OperatorHub*.
+
+. Type *Kiali* into the filter box to find the Kiali Operator.
+
+. Click the *Kiali Operator* provided by Red Hat to display information about the Operator.
+
+. Click *Install*.
+
+. On the *Create Operator Subscription* page, select *All namespaces on the cluster (default)*. This installs the Operator in the defaut `openshift-operators` project and is made available to all projects in the cluster.
+
+. Select the *stable* Update Channel.
+
+. Select the *Automatic* Approval Strategy.
++
+[NOTE]
+====
+The Manual approval strategy requires a user with appropriate credentials to approve the Operator install and subscription process.
+====
+
+. Click *Subscribe*.
+
+. The *Subscription Overview* page displays the Kiali Operator's install process.
+
+
+[id="ossm-operator-install-istio_{context}"]
+== Installing the {ProductName} Operator
+
+.Prerequisites
+
+* Access to the {product-title} web console.
+* The Elasticsearch Operator must be installed.
+* The Jaeger Operator must be installed.
+* The Kiali Operator must be installed.
+
+.Procedure
+
+. Log into the {product-title} web console.
+
+. Navigate to *Catalog* -> *OperatorHub*.
+
+. Type *{ProductName}* into the filter box to find the {ProductName} Operator.
+
+. Click the {ProductName} Operator to display information about the Operator.
+
+. Click *Install*.
+
```

Comment:
Missing this information that exists for the other operators:

"On the *Create Operator Subscription* page, select *All namespaces on the cluster (default)*. This installs the Operator in the defaut `openshift-operators` project and is made available to all projects in the cluster."

Reply:
Fixed.